tests/test_in_toto_mock.py

```diff
@@ -58,13 +52,12 @@ def setUpClass(self):
 
   @classmethod
   def tearDownClass(self):
-    """Change back to initial working dir and remove temp test directory. """
-    os.chdir(self.working_dir)
-    shutil.rmtree(self.test_dir)
+    self.tear_down_test_dir()
 
     # Restore log level to what it was before running in-toto-mock
     logger.setLevel(self._base_log_level)
 
+
```

Comment:
I think there's been some inconsistency here. As I understand it, methods within classes have one blank line before and after. I think we have a mix of single lines and double lines throughout the codebase, and probably something a linter will catch?

Reply:
You're right. Seems like pylint does not support that though. Before we investigate other linters, we should overhaul our code style guidelines. IMO the current recommendations for blank lines (5 lines between top-level functions) is overkill. In in-toto we usually use 2 or 3. I think a slight inconsistency is not a very big deal.